hdtv/plugins/fitInterface.py

```diff
@@ -345,7 +359,7 @@
             fitlist.extend(peaklist)
         return (fitlist, params)
 
-    def ExtractIntegrals(self, fits, integral_type='auto'):
+    def ExtractIntegrals(self, fits, integral_type="auto"):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)